### DISCUSSION TITLE
Prepare an anaglyph in GRASS GIS with aerial images

### DISCUSSION THREAD

**Discussion by JairitoMichael** [OSGeo/grass#3747] [Q&A]
Hello everyone, you are new to using GRASS GIS, and I want to know if there is a way to create anaglyphs, and if so, please help me by explaining the step by step process for creating them.
Kind regards.

**Comment by neteler** [comment]
You may find some related information in the Wiki:

![image](https://github.com/OSGeo/grass/assets/1295172/25f0fa54-2b4a-4636-8b2e-f16347979fba)

There is the very old GRASS GIS 6 script for this purpose (a shell script which may still be usable in GRASS GIS 8 with some modifications):
- https://grasswiki.osgeo.org/wiki/Stereo_anaglyphs

A alternative is to use Paraview:
- https://grasswiki.osgeo.org/wiki/GRASS_and_Paraview#Shown_as_stereo_anaglyphs

Please report your experience then so that we/you can update the Wiki page(s).
